README.md

```diff
@@ -1,2 +1,35 @@
 # piisa
-Personal information identification standard
+
+PIISA stands for a *Personal Identifiable Information Standard*. This standard allows seamless interoperability between various PII detection frameworks.
+
+## Specification
+
+[Click here for the latest specification document](docs/specs.md).
+
+## Who are we
+
+We are a team of privacy enthusiasts who are interested in improving de-identification across multiple domains. 
```

Comment:
maybe something in the lines of _PII management_ would be more encompassing than "de-identification" (which is a key application, granted, but need not be the only one)

Or _PII management, including de-identification_, which has it all :-)

Reply:
Good idea. I'll change it

Reply:
Changed. Could you please re-review and re-approve?